front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -112,14 +113,19 @@ const ProjectsShowPage = memo<Props>(({ project, scrollToEventId }) => {
     return anyIsUndefined(locale, appConfig, project, phases, events);
   }, [locale, appConfig, project, phases, events]);
 
+  // Check that all child components are mounted
+  useEffect(() => {
```

Comment:
This looks hacky 🤔 Not sure if we have any alternatives

Reply:
Nothing at the moment. If it is recommended, then we can go with this. It is a good thing that we don't have many props for this component